docs/conf.py

```diff
@@ -54,18 +54,19 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'pysatSpaceWeather'
-copyright = u'2020, Angeline G Burrell, Jeffrey Klenzing, Russell Stoneback'
-author = u'Angeline G Burrell, Jeffrey Klenzing, Russell Stoneback, Asher Pembroke, Carey Spance'
+project = 'pysatSpaceWeather'
+proj_desc = '{:s} Documentation'.format(project)
```

Comment:
```suggestion
proj_desc = '{:s} Documentation'.format(project)
description = 'Tools for space weather indices.'
```

Reply:
We've been using `title` instead of `proj_desc` in the other packages, but open to changes elsewhere